.eslintrc.js

```diff
@@ -1,224 +1,208 @@
 module.exports = {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true
-    },
-    "extends": [
-        "plugin:@typescript-eslint/recommended",
-        "plugin:@typescript-eslint/recommended-requiring-type-checking",
-        "prettier"
+  env: {
```

Comment:
```suggestion
  root: true,
  env: {
```
so that any eslint configs in a parent directory aren't factored in